recipe/test_blas.sh

```diff
@@ -47,21 +50,14 @@ fi
 
 if [[ "${blas_impl}" == "openblas" && "$(uname)" == MINGW* ]]; then
   # I'm not sure why these fail only on Windows. Skip for now.
+  # Still present as of openblas 0.3.16
   SKIP_TESTS="${SKIP_TESTS}|LAPACK-xeigtsts_sgd_in|LAPACK-xeigtsts_glm_in|LAPACK-xeigtsts_gsv_in|LAPACK-xeigtsts_lse_in|LAPACK-xeigtstd_dgd_in"
   SKIP_TESTS="${SKIP_TESTS}|LAPACK-xeigtstd_glm_in|LAPACK-xeigtstd_gsv_in|LAPACK-xeigtstd_lse_in|LAPACK-xlintstc_ctest_in|LAPACK-xlintstrfc_ctest_rfp_in"
   SKIP_TESTS="${SKIP_TESTS}|LAPACK-xeigtstc_sep_in|LAPACK-xeigtstc_se2_in|LAPACK-xeigtstc_ced_in|LAPACK-xeigtstc_cgd_in|LAPACK-xeigtstc_csb_in"
   SKIP_TESTS="${SKIP_TESTS}|LAPACK-xeigtstc_csg_in|LAPACK-xeigtstc_glm_in|LAPACK-xeigtstc_gsv_in|LAPACK-xeigtstc_lse_in|LAPACK-xeigtstz_zgd_in"
   SKIP_TESTS="${SKIP_TESTS}|LAPACK-xeigtstz_glm_in|LAPACK-xeigtstz_gsv_in|LAPACK-xeigtstz_lse_in|BLAS-xblat1c"
 fi
```

Comment:
@martin-frbg 
Not sure you were aware of this, but there's currently a bunch of tests that fail on openblas & windows (in the peculiar way that conda-forge builds this blas-metapackage).

Hoping to get the lapack tests in https://github.com/conda-forge/openblas-feedstock/pull/122 running for windows as well, then we should see how much of that is "native" (and can hopefully be fixed eventually).

Nothing for you to do right now, just FYI.

Reply:
No, not aware of this. This is specifically with mingw gcc/gfortran and has been happening for a number of previous releases as well ? And do the failures maniffest as "failed to pass the threshold" or do the test codes crash ?

Reply:
I had let everything run through on a debug commit that I rebased out, but it seems Azure deleted the logs already. I'll let it run again.

This is indeed with mingw + gcc (in contrast to the openblas recipe), which might already explain the differences. @isuruf knows this better than I do though, that part of the recipe here hasn't been touched in quite a while.

Reply:
Error log is [here](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=352310&view=logs&j=e5cdccbf-4751-5a24-7406-185c9d30d021&t=66e4a587-a723-53e9-873c-6c9a2c5971ea). From what I can see, mostly segfaults, illegal arguments and the like.

Could be something related to libomp / libgomp again?

It's worth noting that this runs the vanilla netlib test suite (for 3.9.0) but linked against openblas. Not sure if there are any relevant divergences there.

Reply:
Errors of the form ` *** Illegal value of parameter number  1 not detected by` can be ignored, because the way the lapack tests are set up, they need to override the implementation of `xerbla_` in the shared library. In Linux and OSX, the local copy in the test override the `xerbla_` implementation in the shared library, but in windows it doesn't.

Reply:
@isuruf, the thing I don't understand is that - if it was only about xerbla - why is the set of failures not exactly the same between the 4 combinations of {win, osx} + {mkl, openblas}?